src/libcore/slice/mod.rs

```diff
@@ -5773,6 +5773,25 @@ unsafe impl<'a, T> TrustedRandomAccess for RChunksExactMut<'a, T> {
 /// assert_eq!(slice[0], 42);
 /// ```
 ///
+/// ## Counterexample
```

Comment:
"Counterexample" is a weird term here -- that indicates that a theorem or general statement of some sort was made and this example proves that theorem/statement wrong. But that's not what happens.

In the [`MaybeUninit` docs](https://doc.rust-lang.org/nightly/core/mem/union.MaybeUninit.html), we have these in the "Exmaples" section as well, and but we call them "incorrect example" (with emphasis) and also point out clearly in the code the place where we get Undefined Behavior.

Reply:
You're right; I wish there was a word for "Example of what _not_ to do", but since there doesn't seem to be one, I'll go with "Incorrect example"